### PR TITLE
Added configuration for sequelize with postgres for heroku

### PR DESCRIPTION
**Pull request by cryptotearer**
## What does this PR do?
- It is prepared to allow heroku postgres database to work

## What does it contain?
- A Procfile
- .squelizerc file
- config.js (to allow the heroku DATABASE_URL  from the heroku env variables to function)

Postgres-Handles-More-than-You-Think-1569306137588

### CODE REVIEW

**Review by Emmanuel262 — COMMENTED** [review]
- Have you tested the functionality for this configurations or not. It will be great if you had tested.  
- Does this work being blocked by other unfinished work which needs to be done first before Approval?

File: seeders/20220216081856-demo-user.js
Comment (malaba6) [inline seeders/20220216081856-demo-user.js]: We don't need more than two default users, please remove extra users

File: config/config.js
Comment (malaba6) [inline config/config.js]: In development and testing environments, U can leave what was there, U need DATABASE_URL only in production

**Review by malaba6 — CHANGES_REQUESTED** [review]
Did you also add the postgress database to heroku? if not, have a sync with Emmanuel so u can add there.

File: seeders/20220216081856-demo-user.js
Comment (cryptotearer) [reply seeders/20220216081856-demo-user.js]: Yes I did,  in collaboration with Emmanuel 3 days ago

Comment (cryptotearer) [reply seeders/20220216081856-demo-user.js]: And I am going to go ahead and remove them

File: config/config.js
Comment (cryptotearer) [reply config/config.js]: Okay, great!

**Review by malaba6 — APPROVED** [review]

**Pull request merged**
